### DISCUSSION TITLE
How to aprove Request without sending a hash?

### DISCUSSION THREAD

**Discussion by paddyfink** [WalletConnect/walletconnect-monorepo#1866] [Developer Support]
Hello,

How can I call the function ` approveRequest` without providing a hash? I"m sending the transaction via a multi-sig wallet (gnosis) and a hash is not available yet when the function is called.

According to the doc, the function should return the transaction hash, or the zero hash if the transaction is not yet available.

What is the `zero hash` ?
I tried `0x000000000000000000000000000000000000000000000000000000000000000`, `0x`, `0x0`...

**Comment by fereshteh-jvz** [comment]
Have you got your answer? I have the same problem.. Also when I return the transaction hash, UniSwap does not show any reaction.
